content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -380,7 +380,8 @@ you can use to more precisely control the behavior of startup, liveness and read
 checks:
 
 * `initialDelaySeconds`: Number of seconds after the container has started
-before startup, liveness or readiness probes are initiated. Defaults to 0 seconds. Minimum value is 0.
+before startup, liveness or readiness probes are initiated. If a startup probe is defined, liveness and readiness probe delays do not begin until the startup probe has succeeded.
```

Comment:
Could this line be wrapped like all the others? It helps our localizations when translating, thanks.

Reply:
Done, I've also wrapped some other long lines.

What is the recommended line length? I could note it in the Documentation Style Guide.

Reply:
The usual guidance followed is approximately 100 characters per line – a bit above and below is okay if needed 👍